src/styles/utilies/_extend.scss

```diff
@@ -0,0 +1,8 @@
+%position_border-radius {
+  position: absolute;
+  // left: 50%;
```

Comment:
Don't leave commented code.

Reply:
Thanks, will add changes accodringly to your advices!